### PR TITLE
Add missing code snippets for PF5 alpha move

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: _posts/2023-05-03-patternfly-v5-alpha.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+<b>Cockpit has decided to go ahead with an early adoption of PatternFly 5.</b>The motivation for adopting PatternFly 5 can be summarized in several key points. Firstly, the early adoption of PatternFly 5 is advantageous for the PatternFly team. Secondly, as PatternFly 5 is still in its alpha phase, it is more receptive to significant changes. Thirdly, the high code coverage of Cockpit, with its integration and image comparison testing, provides confidence in being one of the early adopters without compromising product quality. Finally, PatternFly 4 is in development freeze, which hinders the ability to utilize the latest features and fixes found in PatternFly 5.
```
Comment (martinpitt) [inline _posts/2023-05-03-patternfly-v5-alpha.md:5]: Please don't use HTML markup in Markdown. `**` was correct, but it lacked a space after the second `**`.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Can you please fix the broken "Upgrading to PatternFly 5" heading as well? It's also wrong to use bold text in a heading, let's leave the standard heading formatting.

**Comment by KKoukiou** [comment]
Preview looks good now https://github.com/KKoukiou/cockpit-project.github.io/blob/main/_posts/2023-05-03-patternfly-v5-alpha.md

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**